javascript/javascript1/week4/palindromic_string/pstring.js

```diff
@@ -0,0 +1,32 @@
+function longestPalindrome(str) {
+  const string = str.toLowerCase();
+  if (str.length < 2) {
+    return null;
```

Comment:
better to return "" (empty string) instead of null, because your valid results are also strings.